core/arch/arm/kernel/thread_a64.S

```diff
@@ -225,7 +225,7 @@ END_FUNC thread_unwind_user_mode
 	.endm
 
 #define INV_INSN	0
-FUNC thread_excp_vect , : align=2048
+FUNC thread_excp_vect , : , default, 2048, naked
```

Comment:
Isn't it possible to keep the syntax `param=value` for `align=2048`, and possibly for other parameters too? It is more comfortable to read.

`s/naked/nobti/` and same below.

Reply:
I had tried various combinations for it but none could work
FUNC thread_excp_vect , :  align=2048 _bti=nobti   -> This resulted in a new section getting created called _bti=nobti
FUNC thread_excp_vect , : , align=2048 _bti=nobti -> Compilation error
FUNC thread_excp_vect , :  section=default align=2048 _bti=nobti -> Compilation error

Reply:
Missing commas perhaps?